pkg/controller/networkpolicy/external_entity.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2020 Antrea Authors
```

Comment:
I didn't compare line by line, just want to check whether this is a simple code move or there's code change as well.

Reply:
This file was created in 2020 anyways...

Reply:
I mean whether the content of these files are changed by this PR or it's just moved here? The review panel doesn't provide difference because it's in a different file from its original place.

Reply:
It was moved w/o any code change